### PR TITLE
Restructure the docs to use Vanilla docs layout

### LINKED ISSUE #8874: Icon for external link for a png seems odd
imageremotelink

The fact that there is an icon for an "external link" attached to a png seems odd to me.
---
*Reported from: https://ubuntu.com/ceph/docs*

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Build two central docs templates (discourse or markdown) with Vanilla docs layout
- Remove all the other docs templates
- Set all docs pages to use the new templates
- Update the docs to use `h3` instead of `strong` titles in the navigation
- As this changes the main content from a 12 column layout to a 9 column I had to change a few classes in the markdown
- Removed the old docs layout styling
- Made the ceph navigation persistent as you navigate the subpages

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- There are four docs (2 from discourse and 2 as markdown). Please check through the following sections:
  - /kubernetes/docs (markdown)
  - smartstart/guide (markdown)
  - server/docs (discourse)
  - ceph/docs (discourse)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8856
Fixes #8874

### CODE REVIEW

File: templates/kubernetes/docs/1.18/charm-containerd.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ juju run-action containerd ACTION [parameters] [--wait]
       debug
     </h5>
   </div>
-  <div class="col-7">
+  <div class="col-5">
```
Comment (anthonydillon) [inline templates/kubernetes/docs/1.18/charm-containerd.md:126]: @evilnick note I needed to tweak the col count here as the docs main content now has a total of 9. Would you mind updating the generation script please?

File: templates/templates/docs/discourse.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
+
+{% extends "templates/base.html" %}
+
+{% block title %} {{ document.title }} | Server documentation{% endblock %}
```
Comment (pmahnke) [inline templates/templates/docs/discourse.html:4]: not that I have looked at all the code, but should ' | Server documentation' be in this file?

Comment (anthonydillon) [reply templates/templates/docs/discourse.html]: Your right, copy and paste error. Done

**Review by pmahnke — APPROVED** [review]
+1, looks much better

**Comment by anthonydillon** [comment]
@pmahnke good to go? can you update the labels please

**Comment by pmahnke** [comment]
@anthonydillon fine by me, I was going to let @evilnick comment... But I gave you greens

**Comment by evilnick** [comment]
Yeah, I can change the template if it's just that :+1:

**Review by evilnick — APPROVED** [review]

**Comment by pmatulis** [comment]
The Ceph docs menu has taken on some odd rendering. It looks like the first occurrence of a multi-level menu item has the parent heading mis-aligned (to the left):

image

**Comment by anthonydillon** [comment]
Thanks for reporting that @pmatulis. I've fixed the ceph docs by making the sub section a strong element. To match the styling required by [the new docs layout](https://vanillaframework.io/docs/layouts/documentation#example).

**Comment by pmatulis** [comment]
@anthonydillon I cannot find where the layout says I need to use bold in these situations but the menu looks much better now. Thank you.

**Comment by anthonydillon** [comment]
I view-source in the example. It planned to tighten up the structure of this side navigation in time so will be more clear in the future.

**Pull request merged**